packages/airnode-node/src/evm/wallet.ts

```diff
@@ -37,6 +37,10 @@ export function getAirnodeWallet(config: Config): ethers.Wallet {
   return ethers.Wallet.fromMnemonic(mnemonic);
 }
 
+export function getAirnodeWalletWithPrivateKey(privateKey: string): ethers.Wallet {
```

Comment:
Do we still need the above `getAirnodeWallet` function?

Reply:
Yep, that's still needed in the `airnode-deployer` to get the wallet from which to get the `privateKey` from initially. It is also used in `airnode-examples` in a few places.

Reply:
No, using `getAirnodeWallet` is fine, I'm just making sure there's no unsued function :slightly_smiling_face: